### PR TITLE
resources: http: Start implementing the http resource

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

File: resources/http.go [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+	// URL is the url targetted by this resource.
+	URL string `yaml:"url"`
+
+	// Method is the method to use: fetch or push.
+	// The fetch method will retrieve data with a GET method,
+	// and use the send/recv mechanism.
+	// The push method will retrieve data, and POST
+	// it as soon as it differs.
+	Method string `yaml:"lens"`
```
Comment (purpleidea) [inline resources/http.go]: lens

**Comment by purpleidea** [comment]
When you're ready for a merge, ping me!

File: resources/http.go [hunk 2]
```diff
@@ -22,6 +22,7 @@ import (
 	"fmt"
 
 	"github.com/purpleidea/mgmt/event"
+	"github.com/r3labs/sse"
```
Comment (purpleidea) [inline resources/http.go]: What's this about? Don't we just want the plain golang http stuff?

**Comment by roidelapluie** [comment]
ENOTIME

**Pull request closed without merging**